ontologies/semopenalex-ontology.ttl

```diff
@@ -0,0 +1,527 @@
+@prefix : <https://semopenalex.org/vocab#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix pmc: <https://www.ncbi.nlm.nih.gov/pmc/articles/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix fn: <http://www.w3.org/2005/xpath-functions#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix cito: <http://purl.org/spar/cito/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix foaf: <http://xmlns.com/foaf/0.1/> .
+@prefix prism: <http://prismstandard.org/namespaces/basic/2.0/> .
+@prefix rdf4j: <http://rdf4j.org/schema/rdf4j#> .
+@prefix wgs: <http://www.w3.org/2003/01/geo/wgs84_pos#> .
+@prefix org: <http://www.w3.org/ns/org#> .
+@prefix gn: <http://www.geonames.org/ontology#> .
+@prefix fabio: <http://purl.org/spar/fabio/> .
+@prefix Default: <http://www.metaphacts.com/resource/> .
```

Comment:
do you many want to remove "Default" and "sesame" prefixes? as well as rdf4j? Please double check the ones that are really relevant